hnn_core/cell.py

```diff
@@ -533,31 +535,53 @@ def create_tonic_bias(self, amplitude, t0, tstop, loc=0.5):
         stim.amp = amplitude
         self.tonic_biases.append(stim)
 
-    def record_soma(self, record_vsoma=False, record_isoma=False):
-        """Record current and voltage at soma.
+    def record(self, record_vsoma=False, record_isoma=False,
+               record_vsec=False, record_isec=False):
+        """ Record current and voltage from all sections
 
         Parameters
         ----------
         record_vsoma : bool
-            Option to record somatic voltages from cells
+            Option to record somatic voltages from cells. Default: False.
         record_isoma : bool
-            Option to record somatic currents from cells
-
+            Option to record somatic currents from cells. Default: False.
+        record_vsec : bool
```

Comment:
voltage_secs : 'soma' | 'all' | list

? But you decide, I don't want to add unnecessary complexity

Reply:
What do you think of `record_vsec : 'soma' | 'all' | None`?

I'm all for it, but if we propagate this change to `simulate_dipole()` it'd be an API change. I don't know of anyone (besides me) using this functionality in `hnn-core` so a deprecation warning seems unnecessary, but worth bringing up here.

Reply:
Let me know what you think! I think the code definitely reads better without lugging around 4 parameters everywhere